server/src/main/java/org/elasticsearch/action/admin/indices/analyze/DetailAnalyzeResponse.java

```diff
@@ -251,7 +233,7 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
-    public static class AnalyzeTokenList implements Streamable, ToXContentObject {
+    public static class AnalyzeTokenList implements Writeable, ToXContentObject {
```

Comment:
should we make `name` and `tokens` fields to be `final`?

Reply:
++